SpecificModConfigMenu.cs

```diff
@@ -370,48 +371,49 @@ public override void update(GameTime time)
+                b.Draw(Game1.staminaRect, new Rectangle(0, 0, Game1.uiViewport.Width, Game1.uiViewport.Height), new Color(0, 0, 0, 192));
 
-                int boxX = (Game1.viewport.Width - 650) / 2, boxY = (Game1.viewport.Height - 200) / 2;
+                int boxX = (Game1.uiViewport.Width - 650) / 2, boxY = (Game1.uiViewport.Height - 200) / 2;
                 IClickableMenu.drawTextureBox(b, boxX, boxY, 650, 200, Color.White);
 
                 string s = "Rebinding key: " + keybindingOpt.Name;
                 int sw = (int)Game1.dialogueFont.MeasureString(s).X;
-                b.DrawString(Game1.dialogueFont, s, new Vector2((Game1.viewport.Width - sw) / 2, boxY + 20), Game1.textColor);
+                b.DrawString(Game1.dialogueFont, s, new Vector2((Game1.uiViewport.Width - sw) / 2, boxY + 20), Game1.textColor);
 
                 s = "Press a key to rebind";
                 sw = (int)Game1.dialogueFont.MeasureString(s).X;
-                b.DrawString(Game1.dialogueFont, s, new Vector2((Game1.viewport.Width - sw) / 2, boxY + 100), Game1.textColor);
+                b.DrawString(Game1.dialogueFont, s, new Vector2((Game1.uiViewport.Width - sw) / 2, boxY + 100), Game1.textColor);
             }
 
             if ( keybinding2Opt != null )
             {
-                b.Draw( Game1.staminaRect, new Rectangle( 0, 0, Game1.viewport.Width, Game1.viewport.Height ), new Color( 0, 0, 0, 192 ) );
+                b.Draw( Game1.staminaRect, new Rectangle( 0, 0, Game1.uiViewport.Width, Game1.uiViewport.Height ), new Color( 0, 0, 0, 192 ) );
 
-                int boxX = (Game1.viewport.Width - 650) / 2, boxY = (Game1.viewport.Height - 200) / 2;
+                int boxX = (Game1.uiViewport.Width - 650) / 2, boxY = (Game1.uiViewport.Height - 200) / 2;
                 IClickableMenu.drawTextureBox( b, boxX, boxY, 650, 200, Color.White );
 
                 string s = "Rebinding key: " + keybinding2Opt.Name;
                 int sw = (int)Game1.dialogueFont.MeasureString(s).X;
-                b.DrawString( Game1.dialogueFont, s, new Vector2( ( Game1.viewport.Width - sw ) / 2, boxY + 20 ), Game1.textColor );
+                b.DrawString( Game1.dialogueFont, s, new Vector2( ( Game1.uiViewport.Width - sw ) / 2, boxY + 20 ), Game1.textColor );
 
                 s = "Press a key combination to rebind";
                 sw = ( int ) Game1.dialogueFont.MeasureString( s ).X;
-                b.DrawString( Game1.dialogueFont, s, new Vector2( ( Game1.viewport.Width - sw ) / 2, boxY + 100 ), Game1.textColor );
+                b.DrawString( Game1.dialogueFont, s, new Vector2( ( Game1.uiViewport.Width - sw ) / 2, boxY + 100 ), Game1.textColor );
             }
 
             drawMouse(b);
 
             if (Constants.TargetPlatform != GamePlatform.Android)
             {
+                Mod.instance.Helper.Events.Input.ButtonReleased += CheckEscape;
```

Comment:
This addition allows user to exit from "Specific Mod Config" menu using the Escape button.